composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "laravel/socialite",
+    "name": "tevfikmert/socialite",
```

Comment:
Please don't change the name of the package. Note also, that it not recommended practice to publish forks on packagist.